src/WebJobs.Script/ExtensionRequirements/ExtensionRequirementsInfo.cs

```diff
@@ -12,10 +12,10 @@ internal sealed class ExtensionRequirementsInfo
         public ExtensionRequirementsInfo(IEnumerable<BundleRequirement> bundleRequirements, IEnumerable<ExtensionStartupTypeRequirement> extensionRequirements)
         {
             BundleRequirementsByBundleId = bundleRequirements
-                .ToDictionary(a => a.Id, StringComparer.OrdinalIgnoreCase);
+                ?.ToDictionary(a => a.Id, StringComparer.OrdinalIgnoreCase);
 
             ExtensionRequirementsByStartupType = extensionRequirements
-                .ToDictionary(a => a.Name, StringComparer.OrdinalIgnoreCase);
+                ?.ToDictionary(a => a.Name, StringComparer.OrdinalIgnoreCase);
```

Comment:
nit: it's odd to see the line start with `?`; either update for these to be on one line. Or you could start the method with a null check on `bundleRequirements`

Reply:
have fixed this